packages/integration-tests/api/api.js

```diff
@@ -394,6 +376,134 @@ async function addProviderApi({ prefix, provider }) {
   });
 }
 
+/**
+ * Fetch a list of providers from the Cumulus API
+ *
+ * @param {Object} params - params
+ * @param {string} params.prefix - the prefix configured for the stack
+ * @returns {Promise<Object>} - the list of providers fetched by the API
+ */
+async function getProviders({ prefix }) {
```

Comment:
Nitpicky, but it'd be nice to get some of these into separate files, like a workflows file. This one is getting too big. I won't block your merge on it though.